src/connect_dialog_handler.rs

```diff
@@ -1,8 +1,8 @@
 #![allow(unused)]
 
 use crate::connect_dialog_handler::ConnectDialogMessage::{
-    ConnectButtonPressed, ConnectionError, HideConnectDialog, ModalKeyEvent,
-    NodeIdEntered, RelayURL, ShowConnectDialog,
+    ConnectButtonPressed, ConnectionError, HideConnectDialog, ModalKeyEvent, NodeIdEntered,
+    RelayURL, ShowConnectDialog,
 };
```

Comment:
still getting these reformatting changes.... what has changed that is causing this do you think?

Reply:
Whatever comes out after running rustfmt is the correct formatting, and we should both make sure our setups produce that.